src/components/SignUp/SignUp.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import { Link, useHistory } from 'react-router-dom';
+import { useFirebase } from 'react-redux-firebase';
+import LabeledField from '../_common/LabeledField/LabeledField';
```

Comment:
better `import { LabeledField } from '../_common/LabeledField'` (index.js), avoid default exports :)

https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad